devito/passes/clusters/blocking.py

```diff
@@ -257,11 +255,14 @@ def callback(self, clusters, prefix):
                 return clusters
             skew_dim = skew_dims.pop()
 
+            # The level of a given Dimension in the hierarchy of block Dimensions
```

Comment:
Instead of writing "what level is", could you write "why" ? what is its purpose?